app/main.py

```diff
@@ -1 +1,46 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(
+            self,
+            name: str,
+            health: int = 100,
+            hidden: bool = False
+    ) -> None:
+
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        if self.health > 0:
+            Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        return (f"{{"
+                f"Name: {self.name}, "
+                f"Health: {self.health}, "
+                f"Hidden: {self.hidden}"
+                f"}}")
```

Comment:
The `__repr__` method uses inconsistent quotes. Ensure that double quotes are used consistently throughout the code as per the style guidelines.